python_modules/dagster/dagster/_grpc/server.py

```diff
@@ -364,7 +372,9 @@ def __init__(
             if not lazy_load_user_code:
                 raise
             self._loaded_repositories = None
-            self._serializable_load_error = serializable_error_info_from_exc_info(sys.exc_info())
+            self._serializable_load_error = serializable_error_info_from_exc_info(
+                sys.exc_info(), hoist_user_code_execution_error=True
```

Comment:
whats up with this change?

Reply:
Since we introduce a `user_code_error_boundary` in the code location load path above, the error that gets bubbled up to this point is now a `DagsterUserCodeLoadError` rather than the underlying error message. The underlying msg is still available in the stack, so I think it's fine - this caused a few tests to fail, but changed to just updating the tests and removing this.

Reply:
Its worth doing if it meaningfully improves user experience, maybe worth including some screenshots in the PR to demonstrate